R/Driver.R

```diff
@@ -37,48 +37,63 @@ setMethod(
     # TODO: Print more details
   })
 
-#' Connect to a ODBC compatible database
+#' Connect to an ODBC compliant database
```

Comment:
I wonder if it should actually be "Connect to a database via an ODBC driver"?  I think it's important to mention the driver here, because that's what's ODBC compliant, not the database.